config/Dockerfiles/fedoraci-runner/package-test/package-test.sh

```diff
@@ -175,6 +176,34 @@ if [[ `ls -1 tests*.yml 2>/dev/null | wc -l` == 0 ]]; then
     exit 1
 fi
 
+# Merge two results.yml files.
+#
+# This function merges previous results.yml into current results.yml.
+# If the two results.yml files are the same, it does nothing.
+#
+# Arguments:
+# - path to the previous results.yml file (e.g.: ${TEST_ARTIFACTS}/results.yml.prev)
+# - path to the current results.yml file
+function merge_results {
```

Comment:
The 'function' keyword is unnecessary.

Reply:
Yeah, you're right of course - I was following the convention used in the rest of the file here. Should I keep it here, or drop it? :)

Reply:
I did notice it was the convention in the rest of the file after I did this review.  My personal opinion is that we should not use this bashism in shell scripts.  We should write shell functions as:

function_name() { }

I would leave it as-is in this patch and submit a separate PR to change the way functions are written.

Reply:
Agreed. Let's do this in a separate PR.